ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -48,6 +49,7 @@ def __init__(self):
         # TODO: Add other member variables you need below
         self.pose = None
         self.wps = None
+        self.wps_base = None
```

Comment:
Need to add a check in `pose_cb` - `        if self.wps and self.wps_base:`

Otherwise, there can be a race condition where `waypoints_cb` is not yet complete and yet, we enter `pose_cb` and try to use `wps_base`

Reply:
agree